File: C#/ConvertCamel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

public class ConvertCamel
{
  
 public static string ToCamelCase(string str)
        {
            string[] wordArray = str.Replace("_","-").Split("-");
            char[] characterArray;
            for (int li = 1; li < wordArray.Length; li++)
            {
                characterArray = wordArray[li].ToArray();
                characterArray[0] = Char.ToUpper(characterArray[0]);
                wordArray[li] = new string(characterArray);
            }
            return string.Join("", wordArray);
        }
}

File: JavaScript/convert-camel.js
function toCamelCase(str){
    let wordArray = str.replace(/_/g,"-");
    var position;
    while (wordArray.indexOf("-") != -1)
    {
        position = wordArray.indexOf("-");
        wordArray = wordArray.replace("-","");
        wordArray= wordArray.substr(0, position) + wordArray.charAt(position).toUpperCase() + wordArray.substr(position + 1, wordArray.length)
    }
    return wordArray;
}